spring-cloud-circuitbreaker-resilience4j/src/main/java/org/springframework/cloud/circuitbreaker/resilience4j/Resilience4JCircuitBreakerFactory.java

```diff
@@ -61,13 +63,15 @@ public Resilience4JCircuitBreakerFactory() {
 	}
 
 	public Resilience4JCircuitBreakerFactory(CircuitBreakerRegistry circuitBreakerRegistry,
-			TimeLimiterRegistry timeLimiterRegistry, Resilience4jBulkheadProvider bulkheadProvider) {
+			TimeLimiterRegistry timeLimiterRegistry, Resilience4jBulkheadProvider bulkheadProvider,
```

Comment:
Add a new constructor

Reply:
Can you clarify why does it need a new constructor? 
This class is in only instantiated from Resilience4JAutoConfiguration.java.

Reply:
It is a public class so anyone can extend it, we try not to break people in minor releases